crates/composefs/src/repository.rs

```diff
@@ -416,15 +416,25 @@ impl<ObjectID: FsVerityHashValue> Repository<ObjectID> {
             Err(other) => Err(other)?,
         }
     }
+
+    pub fn mount(&self, name: &str) -> Result<OwnedFd> {
+        let (image, enable_verity) = self.open_image(name)?;
+        Ok(composefs_fsmount(
+            image,
+            name,
+            self.objects_dir()?,
+            enable_verity,
+        )?)
     }
 
-    pub fn mount(&self, name: &str, mountpoint: &str) -> Result<()> {
-        let image = self.open_image(name)?;
+    pub fn mount_at(&self, name: &str, mountpoint: &str) -> Result<()> {
+        let (image, enable_verity) = self.open_image(name)?;
         Ok(mount_composefs_at(
```

Comment:
Can we maybe get rid of `mount_composefs_at()` in `mount.rs` and use `move_mount(self.mount(name), mountpoint)` here instead (or whatever `mount_at` internal helper we have?)